js/components/close-caption-multi-audio-menu/closeCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,145 @@
+var React = require('react');
+var CloseCaptionTab = require('./closeCaptionTab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var CloseCaptionMultiAudioMenu = React.createClass({
+
+  getInitialState: function () {
+    this.multiAudio = {};
+    this.closeCaptions = {};
+
+    if(this.props.controller && this.props.controller.state) {
+      this.multiAudio = this._getMultiAudio();
+      this.closeCaptions = this._getClosedCaptions();
+    }
+
+    return {
+      multiAudio: this.multiAudio,
+      closeCaptions: this.closeCaptions,
+      skinConfig: this.props.skinConfig
```

Comment:
Copying props to state is usually an anti-pattern. Please use the prop directly if possible